content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -163,6 +163,13 @@ have been frozen starting from September 13, 2023. Please read our
 for more details.
 {{< /note >}}
 
+{{< note >}}
+There's a dedicated package repository for each Kubernetes minor version. If you want to install
+a minor verson different than {{< skew currentVersion >}}, please see the installation guide for
+your desired minor version. The new package repositories support Kubernetes releases starting
+with v1.24.0. 
```

Comment:
Can you share some more specific details from where the user can find the installation guide for their desired minor version.

Reply:
Fixed in the latest commit